src/interfaces.d.ts

```diff
@@ -414,6 +455,7 @@ export interface NodeHandlerInterface extends Evented {
 export interface WidgetMetaProperties {
 	invalidate: () => void;
 	nodeHandler: NodeHandlerInterface;
+	bind: any;
```

Comment:
Is this really `any`?

Reply:
the bind scope is any in `widgetBase` I believe? I guess it should probably be `WidgetBaseConstructor` ?

Reply:
It wouldn't be the constructor, it would just be `WidgetBase` if it is able to be typed.

Reply:
ok